github-users-web-client/pom.xml

```diff
@@ -113,6 +113,12 @@
           <compilerArgs>-generateJsInteropExports,-optimize,9,-XnoclassMetadata</compilerArgs>
         </configuration>
       </plugin>
+      <plugin>
+        <!-- just here to make IntelliJ happy -->
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>gwt-maven-plugin</artifactId>
+        <version>2.8.1</version>
```

Comment:
I have been making the releases last years. The plugin had not changed anything hehe, so... it is not very important, intellij detect the plugin but uses the dependency correctly to configure the project. So... do whatever you prefer. Probably you end up using gradle so the android project can be built correctly, so this problem will disappear.